dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.webProfile.internal-9.0.feature

```diff
@@ -0,0 +1,19 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=io.openliberty.webProfile.internal-9.0
+singleton=true
+-features=io.openliberty.connectionManager1.0.internal.ee-9.0,\
+io.openliberty.authFilter1.0.internal.ee-9.0,\
+io.openliberty.globalhandler1.0.internal.ee-9.0,\
+io.openliberty.jcaSecurity.internal.ee-9.0,\
+io.openliberty.jdbc4.0.internal.ee-9.0,\
+io.openliberty.jdbc4.2.internal.ee-9.0,\
+io.openliberty.restConnector2.0.internal.ee-9.0,\
+io.openliberty.sessionCache1.0.internal.ee-9.0,\
+io.openliberty.sessionDatabase1.0.internal.ee-9.0,\
+io.openliberty.webBundle.internal.ee-9.0,\
+io.openliberty.webBundleSecurity1.0.internal.ee-9.0,\
+io.openliberty.wimcore.internal.ee-9.0,\
```

Comment:
sessionCache, sessionDatabase, and wim is not part of webProfile.  We will try to load the bundles referenced by these private feature which will cause issues since sessionCache, sessionDatabase, wim bundles are not found.  You have it here in order to get things to be in the image for the webProfile package, but maybe need to reference this feature in the features.xml instead.  It would have to be a public feature though in that case.  Ugh.